src/System.Reflection.Emit/ref/System.Reflection.Emit.csproj

```diff
@@ -3,17 +3,21 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <PropertyGroup>
     <ProjectGuid>{4FBD1556-7B96-414E-9EA4-85281956D60E}</ProjectGuid>
+    <DefineConstants Condition="'$(TargetGroup)' == 'netstandard'">$(DefineConstants);TargetsNetstandard</DefineConstants>
```

Comment:
Either move these to a separate file as @AtsushiKan suggests or change this to follow the convention: https://github.com/dotnet/corefx/blob/6cb23ac20696ab314d2b28e95af40c8454bd7c0d/Documentation/coding-guidelines/project-guidelines.md#define-naming-convention.

Reply:
I'll take care of this.